docs/en/ingest-management/elastic-agent/scaling-on-kubernetes.asciidoc

```diff
@@ -0,0 +1,290 @@
+[[scaling-on-kubernetes]]
+= Scaling {agent} on kubernetes
+
+For more information on how to deploy Elastic Agent on Kubernetes please review the pages below:
+
+- How to run the {agent} on <<running-on-kubernetes-managed-by-fleet>>.
+- How to run the {agent} on <<running-on-kubernetes-standalone>>.
+
+[discrete]
+== Observability at scale
+
+This document will try to summarize the key factors and best practices for using https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-kubernetes.html[Elastic Observability] to monitor Kubernetes infrastructure at scale. Users need to consider different parameters and adjust Elastic Stack accordingly. The following elements are being affected as the size of Kubernetes cluster increases:
+
+- The amount of metrics being collected from several kubernetes endpoints
+- The Agent's resources to cope with the high CPU and Memory needs for the internal processing
+- The ElasticSearch resources needed due to the higher rate of metric ingestion
+- The Dashboard's Visualisations response times as more data are requested on a given time window 
+
+The document is divided in two main sections:
+
+- <<configuration-practices,Configuration Best Practices>>
+- <<validation-and-troubleshooting-practices,Validation and Troubleshooting methods>>
+
+[discrete]
+[[configuration-practices]]
+== Configuration Best Practices
```

Comment:
```suggestion
== Configuration best practices
```

Reply:
The Elastic writing standard is to use sentence case capitalization (not initial caps)